Jenkinsfile

```diff
@@ -96,8 +96,6 @@ node('ubuntu-zion') {
       
       //Create tar of our image
       dir('build') {
-        OsTools.runSafe(this, "docker save ${imageName}")
```

Comment:
I'd also recommend re-working the other `docker save` call in this file, which uses a redirect to gzip instead of the available `-o` flag. Also - not clear that anything consumes that artifact?

Reply:
The `OsTools.runSafe(this, "docker save ${imageName} | gzip > ${archiveName}.tar.gz")` is from 4 years old and was committed by @bigspotteddog. I will ask him the motivation and if we can remove / replace it. I think we can still merge this PR?

Reply:
Looks to me like the other `docker save` is for creating a Jenkins artifact, which is possibly useful for debugging.

Reply:
> Looks to me like the other `docker save` is for creating a Jenkins artifact, which is possibly useful for debugging.

I can't see many use cases where it would be useful - for someone that has Docker installed and knows how to import tar's but can't build from source? @koraytugay is going to ask and see if there's memory as to why it's there - and in the end its a few GB of file storage anyhow. I was just trying to head off any unintended dump to stdOut if this gets refactored in the future